vcxagency-node/src/index.js

```diff
@@ -121,10 +123,15 @@ validateAppConfig(appConfig, (err, ok) => {
     appAgent.use('/agency/msg', appAgentMsg)
     appAgent.use('/', appAgentJson)
 
-    appAgent.listen(
-      appConfig.SERVER_PORT,
-      () => logger.info(`Agency is listening on port ${appConfig.SERVER_PORT}!`)
-    )
+    if (appConfig.SERVER_ENABLE_TLS === 'true') {
+      https.createServer({
+        cert: fs.readFileSync('certs/agency.pem'),
```

Comment:
should be configurable

Reply:
Can you please be more specific? Do you mean certificate path?

Reply:
Sorry, yeah that's what I meant